resources/helm/dask-gateway/values.yaml

```diff
@@ -126,6 +126,9 @@ gateway:
         request: null
         limit: null
 
+      # Number of threads available for a worker
+      threads: 1
```

Comment:
Can you please explain this? It seems to me reasonable that in real use, multiple threads per worker can be a good thing.

Reply:
It's just a default value for testing, otherwise it would come out to be 0:

```
The value of the 'worker_threads' trait of a KubeClusterConfig instance should not be less than 1, but a value of 0 was specified.
```

In practice you'll override this with your own values, with whatever your requirement is

Reply:
OK, that's fine. 
Can you change the default to also be 1?

Reply:
Hmmm, this looks like a bug actually. By default the worker should use as many threads as cores (which has a pre-configured default). A user shouldn't be forced to configure the number of threads unless they want it to differ from the number of cores. I think we want `worker_threads` to set `allow_none=True`, then fix the logic in the k8s backend to use `worker_cores` if `worker_threads` is None.